site/styles/_carousel.scss

```diff
@@ -97,15 +97,24 @@ s72-carousel {
   @include media-breakpoint-up(lg) {
     flex-direction: row;
     padding: var(--carousel-bottom-gradient-height) 50px 1px;
-  }
 
-  .sponsor {
-    margin-bottom: 8px;
-    @include media-breakpoint-up(lg) {
+    .sponsor-awards-wrapper {
```

Comment:
These selectors should be defined outside of breakpoint mixin inclusion, not inside, because it encourages non-DRY selector duplication (you've defined `.sponsor` in two places).

Reply:
- Changed to single with breakpoint mixin inside